hs_core/views/__init__.py

```diff
@@ -1054,20 +1053,30 @@ def create_resource(request, *args, **kwargs):
             files=resource_files,
             source_names=source_names,
             # TODO: should probably be resource_federation_path like it is set to.
-            fed_res_path = fed_res_path[0] if len(fed_res_path)==1 else '',
+            fed_res_path = fed_res_path[0] if len(fed_res_path) == 1 else '',
             move=(fed_copy_or_move == 'move'),
             content=res_title
     )
 
     try:
-        utils.resource_post_create_actions(request=request, resource=resource, user=request.user,
-                                           metadata=metadata, **kwargs)
+        utils.resource_post_create_actions(request=request, resource=resource,
+                                           user=request.user, metadata=metadata, **kwargs)
     except (utils.ResourceFileValidationException, Exception) as ex:
         request.session['validation_error'] = ex.message
+        ajax_response_data['message'] = ex.message
+        ajax_response_data['status'] = 'success'
+        ajax_response_data['file_upload_status'] = 'error'
```

Comment:
@pkdash setting ajax response data ```status``` to ```success``` while file validation exception raises sounds confusing to me. If there is not much difficulty to modify ajax js call to handle this with a status of ```error```, it will be clearer to return a status of ```error``` here. I see ```file_upload_status``` is set to ```error``` here, so it is OK to keep it the way it is now, but if there is not much trouble, it'd be ideal to return ```status``` as ```error``` as well. Will leave it to your judgement on this one whether further change is needed or not.